### ISSUE TITLE
feat(config): Support kubeconfig

### ISSUE THREAD

**Issue by gaocegege** [ContainerSSH/kubernetes#100]
## Please describe what you would like to see in ContainerSSH

```
kubernetes:
  connection:
    host: your-kubernetes-api-server:6443
    cert: |
      -----BEGIN CERTIFICATE-----
      ...
      -----END CERTIFICATE-----
    key: |
      -----BEGIN RSA PRIVATE KEY-----
      ...
      -----END RSA PRIVATE KEY-----
    cacert: |
      -----BEGIN CERTIFICATE-----
      ...
      -----END CERTIFICATE-----
```

The config is not user-friendly when running outside of kubernetes.

## Please describe the solution you'd like

It will be better to support kubeConfigPath:

```yaml
kubernetes:
    kubeConfigPath: ~/.kube/config
```

## Please describe your use case

A clear and concise description of how you would use this feature.

**Comment by ghost** [comment]
By the looks of it, Kubeconfig support has made it into the Go client library as `clientcmd.BuildConfigFromFlags` so this should be possible.

**Comment by gaocegege** [comment]
Yep I found the implementation in this repo. But it is not used in config

**Comment by gaocegege** [comment]
https://github.com/ContainerSSH/kubernetes/blob/f4b3a275f5773de4d618f6ba5cfa0b6bfd1f27f5/kubeconfig.go#L17

**Comment by ghost** [comment]
That is a very rudimentary implementation from the days where the only code parsing a Kubeconfig file was in the kubectl codebase.
